### PR TITLE
JavaScript capstone project

### PR DESCRIPTION
**Pull request by SamanAtashi**
# JavaScript capstone project: API-based web app
Features are as follows:
- Use JavaScript to make websites dynamic and build basic single-page apps.
- Use ES6 syntax.
- Use ES6 modules.
- Use callbacks and promises.
- Use webpack.
- Apply JavaScript best practices and language style guides in code.
- Use the AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API TVmaze documentation.
- Understand and use JSON.
- Make JavaScript code synchronous.

### CODE REVIEW

File: src/js/comments.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+          </div>
+
+          <div class="all-comments">
+                <span>Comments</span> 
+                <span class="text-secondery mx-2" ><i class="fa fa-comments" ></i></span> 
+                <span class="commentN text-white">${commentNum || ''}</span>
+                <p class="last-com font-italic">Last comment: ${
+  lastComment.creation_date || 'No comment'
+} </p>
```
Comment (hemantso) [inline src/js/comments.js]: - [x] As per the requirements of the project, When the `Comments popup` loads, the web app is expected to retrieve data from the selected API and show details about the selected item. And likewise, retrieve data from the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments. Now we have only updated the number of comments. We should display the comments also. Let's try to fix this issue.

yours
image

Original example
image

File: src/js/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              </div>
+              <div class="description">
+                <h6>Sumary:</h6>
+              </div>
+
+        </li>`;
+};
+
+const putShowInside = async (show, num) => {
+  // bring them in
```
Comment (hemantso) [inline src/js/index.js]: - [x] Kindly try to delete the comments from the `index.js` file.

File: src/sass/style.scss [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+//   font-size: 0.6rem;
+// }
+
+// ul {
+//   width: 100%;
+//   height: 100%;
+// }
+
+// a {
+//   color: rgb(48, 48, 48);
```
Comment (hemantso) [inline src/sass/style.scss]: - [x] Kindly try to delete the commented part from the file.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @SamanAtashi and @andersonlebon, 

Good job so far! 👍 You guys have done superb work. 👏 
### Let me list some of the highlights
- Very organized codebase
- Good use of the Involvement API to record the different user interactions
- Awesome UI
- Good presentation of features of the project
- Good README documentation 
- Usage of Github Pages to deploy the application


There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/sass/style.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
+@import 'bootstrap';
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Condensed:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Tourney:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+// font-family: 'Tourney', cursive;
+// font-family: 'IBM Plex Sans Condensed', sans-serif;
```
Comment (Dipeshtwis) [inline src/sass/style.scss]: - [x] Still, I can see the commented code. Please remove that to make the codebase more cleaner.

File: src/sass/style.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+// *::selection {
+//   background-color: rgba(245, 174, 41, 0.5);
+// }
```
Comment (Dipeshtwis) [inline src/sass/style.scss]: - [x] Same case here, try removing all the commented code for a cleaner codebase.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @SamanAtashi and @andersonlebon, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**